nevergrad/optimization/recastlib.py

```diff
@@ -30,7 +37,7 @@ def _optimization_function(self, objective_function: Callable[[base.ArrayLike],
         # pylint:disable=unused-argument
         budget = np.inf if self.budget is None else self.budget
         best_res = np.inf
-        best_x: np.ndarray = np.zeros(self.dimension)
+        best_x: np.ndarray = self.current_bests["average"].x  #np.zeros(self.dimension)
```

Comment:
@jrapin  this should work, right ?  the goal is that if the algorithm has had tell_not_asked called repeatedly, then we start with the current best.